packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,738 @@
+          <MenuListItem itemId={0} isActive={activeItem === 0}>
+            Action 1
+          </MenuListItem>
+          <MenuListItem itemId={1} isActive={activeItem === 1}>
+            Action 2
+          </MenuListItem>
+          <Divider component="li" />
+          <MenuListItem itemId={2} isActive={activeItem === 2}>
+            Action 3
+          </MenuListItem>
+        </MenuList>
+      </Menu>
+    );
+  }
+}
+```
+
+### With titled groups
+
+```js
+import React from 'react';
+import { Menu, MenuGroup, MenuItem, MenuList, MenuListItem, Divider } from '@patternfly/react-core';
+
+class MenuWithTitledGroups extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = (event, itemId) => {
+      this.setState({
+        activeItem: itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Menu onSelect={this.onSelect}>
+        <MenuGroup>
+          <MenuList>
+            <MenuListItem to="#" itemId={0} isActive={activeItem === 0}>
+              Link not in group
+            </MenuListItem>
+          </MenuList>
+        </MenuGroup>
+        <Divider />
+        <MenuGroup label="Group 1">
+          <MenuList>
+            <MenuListItem to="#" itemId={1} isActive={activeItem === 1}>
+              Link 1
+            </MenuListItem>
+            <MenuListItem itemId={2} isActive={activeItem === 2}>
+              Link 2
+            </MenuListItem>
+          </MenuList>
+        </MenuGroup>
+        <Divider />
+        <MenuGroup label="Group 2">
+          <MenuList>
+            <MenuListItem to="#" itemId={3} isActive={activeItem === 3}>
+              Link 1
+            </MenuListItem>
+            <MenuListItem to="#" itemId={4} isActive={activeItem === 4}>
+              Link 2
+            </MenuListItem>
+          </MenuList>
+        </MenuGroup>
+      </Menu>
+    );
+  }
+}
+```
+
+### With description
+
+```js
+import React from 'react';
+import { Dropdown, Menu, MenuItem, MenuList, MenuListItem } from '@patternfly/react-core';
+import CodeBranchIcon from '@patternfly/react-icons/dist/js/icons/code-branch-icon';
+import LayerGroupIcon from '@patternfly/react-icons/dist/js/icons/layer-group-icon';
+import CubeIcon from '@patternfly/react-icons/dist/js/icons/cube-icon';
+
+class MenuWithTitledGroups extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = (event, itemId) => {
+      this.setState({
+        activeItem: itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Menu onSelect={this.onSelect}>
+        <MenuList>
+          <MenuListItem icon={<CodeBranchIcon />} description="Description" itemId={0} isActive={activeItem === 0}>
+            Action 1
+          </MenuListItem>
+          <MenuListItem
+            isDisabled
```

Comment:
Any interactive items in the list item should be disabled, so the `button.pf-c-menu__item` needs `disabled` here. This is missing in the core workspace, I'll create an issue to update it.